cli/src/base/UploadCommandBase.ts

```diff
@@ -285,6 +285,11 @@ export default abstract class UploadCommandBase extends ContentDirectoryCommandB
     this.log(`Uploading object ${objectId.toString()} (${filePath})`)
     try {
       await axios.post(`${storageNodeInfo.apiEndpoint}/files`, formData, {
+        params: {
+          dataObjectId: objectId.toString(),
+          storageBucketId: storageNodeInfo.bucketId,
+          bagId,
```

Comment:
Forgot to mention in the previous review:
1. You can remove those fields from `formData` now
2. You can update `distributor-node/src/commands/dev/batchUpload.ts` in a similar way

Reply:
Addressed in [b32d4d0](https://github.com/Joystream/joystream/pull/4402/commits/b32d4d0bb53dfb5d4ea80804a85b3b3b58a65776)